catroid/src/main/java/org/catrobat/catroid/ui/MainMenuActivity.java

```diff
@@ -375,16 +387,10 @@ public void handleWebButton(View view) {
 			return;
 		}
 		TrackingUtil.trackStartWebSessionExplore();
-		if (Utils.isUserLoggedIn(this)) {
-			SharedPreferences sharedPreferences = PreferenceManager.getDefaultSharedPreferences(this);
-			String username = sharedPreferences.getString(Constants.USERNAME, Constants.NO_USERNAME);
-			String token = sharedPreferences.getString(Constants.TOKEN, Constants.NO_TOKEN);
 
-			String url = Constants.CATROBAT_TOKEN_LOGIN_URL + username + Constants.CATROBAT_TOKEN_LOGIN_AMP_TOKEN + token;
-			startWebViewActivity(url);
-		} else {
-			startWebViewActivity(Constants.BASE_URL_HTTPS);
-		}
+		String url = Constants.FLAVORED_BASE_URL_HTTPS;
```

Comment:
I've moved that code to a Utils function, as this is used in other places, too.
--> This is now done in: Utils.addUsernameAndTokenInfoToUrl(..)